platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateLifecycles.java

```diff
@@ -34,6 +34,14 @@
  * interface; but in the future will be callbacks registered with a platform builder.
  */
 public interface MerkleStateLifecycles {
+    /**
+     * Called when a {@link MerkleStateRoot} needs to ensure its {@link PlatformStateAccessor} implementation
+     * is initialized.
+     *
+     * @param state the root of the state to be initialized
+     */
+    void initPlatformState(@NonNull State state);
```

Comment:
I love how it's so much easier to reason about `platform state` now that it's just a `state`!